boards/arm/96b_argonkey/Kconfig.defconfig

```diff
@@ -37,4 +37,34 @@ config SPI_2
 
 endif # SPI
 
+if LSM6DSL
+
+choice LSM6DSL_BUS_TYPE
+	default LSM6DSL_SPI
+endchoice
+
+config LSM6DSL_SPI_MASTER_DEV_NAME
+	default "SPI_2"
+
+config LSM6DSL_SPI_SELECT_SLAVE
+	default 1
+
+choice LSM6DSL_TRIGGER_MODE
+	default LSM6DSL_TRIGGER_GLOBAL_THREAD
+endchoice
+
+config LSM6DSL_GPIO_DEV_NAME
+	default "GPIOB"
+
+config LSM6DSL_GPIO_PIN_NUM
+	default 1
+
+config LSM6DSL_SENSORHUB
```

Comment:
warning: LSM6DSL_SENSORHUB (defined at /media/projects/zephyr/boards/arm/96b_argonkey/Kconfig.defconfig:62) defined without a type

Reply:
Yes, Yannick.
Actually those depends by #7028 which is still under discussion.
I guess everything will be merged smoothly. What do you think?
Should I instead remove these two stuff and do another commit later on?
Or maybe split this commit into two, but keeping them in this PR itself?

Reply:
I would mark this PR with a DNM label and wait for #7028 before merging this one.

Reply:
Sure!
I don't think I have permission to do it, have I?
@erwango @galak  can you help me?

Reply:
I added the DNM label.